### PR TITLE
docs(k8s): update k8s docs

### PR DESCRIPTION
**Pull request by rsdmike**
## PR Checklist

- [x] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [x] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

K8s steps/deploiyment guide

## Anything the reviewer should know when reviewing this PR?
This is focused on the technical steps, overview, details, fluff around the steps still needed.

### CODE REVIEW

File: docs/Kubernetes/deployingk8s-aks.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+Login using `az login` if you haven't already and make sure the correct subscription is set as default using `az account set`
+``` bash
+az group create --name <your-resource-group-name> --location eastus
+```
+
+```
+az deployment group create --resource-group <your-resource-group-name> --template-file aks.json
+```
+You will be prompted for a name for the AKS Cluster, the linux user admin name (i.e. your name), and the public key (`.pub` file) you generated in step 1. This takes about ~5 min. Note the `fqdnSuffix` in the "outputs" section of the JSON response (i.e. `westus2.cloudapp.azure.com`) when complete. This will be needed for the updating the commonName in the `values.yaml` file.
```
Comment (kimberlyxdavis) [inline docs/Kubernetes/deployingk8s-aks.md:25]: Hmm. "Step 1". By that do you mean the Create SSH Key?

**Review by kimberlyxdavis — COMMENTED** [review]
I think some formatting stuff needs to happen, but it's a great first draft.

File: docs/Kubernetes/deployingk8s-aks.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+## Prerequisites
+
+- Docker Desktop w/ kubectl
```
[the remark this answers is not in the document]
Comment (rsdmike) [reply docs/Kubernetes/deployingk8s-aks.md:3]: kubektl is the specific command line tool that is used that needs to be used. But essentially the same i guess.

File: docs/Kubernetes/deployingk8s-aks.md (hunk 1 above)
Comment (rsdmike) [reply docs/Kubernetes/deployingk8s-aks.md:25]: Yes, got rid of steps , can change.

[the remark this answers is not in the document]
Comment (rsdmike) [reply docs/Kubernetes/deployingk8s-aks.md:25]: Azure Kubernetes service

**Review by kimberlyxdavis — CHANGES_REQUESTED** [review]

**Review by kimberlyxdavis — APPROVED** [review]

**Pull request merged**